lesson03/assignment/js/index.js

```diff
@@ -0,0 +1,25 @@
+const temp = [];
+
+$.getJSON("https://raw.githubusercontent.com/zyang888/html300/master/lesson03/assignment/data/data.json", function(data) {
+  $.each(data, function(idx, obj){
+          temp.push(obj);
+  });
+  const resHTML = temp.map((person)=>`
+      <article class="main">
+  			<section class="profile">
+  				<img src="./img/unsplash-headshot.jpg" alt="profile-picture">
+  				<h1> ${person.name}</h1>
+  				<p> ${person.jobTitle}</p>
+  			</section>
+  			<section class="info">
+  				<p><b>Company:</b> ${person.company}</p>
+  				<p><b>Experience:</b> ${person.experience}</p>
+  				<p><b>School:</b> ${person.school}</p>
+  				<p><b>Major:</b> ${person.major}</p>
+  				<p><b>Email:</b> ${person.email}</p>
+  				<div><img src="./img/linkedin.svg" alt=""><a href=""> ${person.linkedInUrl}</a></div>
```

Comment:
For the link, we'll want the url to be the href value, not just plain text. You might make the link wrap the logo, so that's what's clickable: 
```
<div><a href="${person.linkedInUrl}">  <img src="./img/linkedin.svg" alt=""> </a></div>
``
Also, great job including an alt tag with empty value for the logo! Accessibility win!